api-reference/listing/sync.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: "Synchronise Listings"
+api: "POST https://api.travtus.com/listing/sync/"
+description: "This endpoint synchronises the listings in the given portfolio with the provided listings."
+---
+
+This endpoint allows you to update, create and delete listings for a portfolio.
+
+The endpoint will attempt to reconcile every listing in the request body with an existing one for the given group and update it with the provided details.
+
+If a listing is not found, a new one will be created.
+
+Listings for the provided portfolio that are not included in the request body will be deleted.
+
+If there are no listings passed in the request body, all listings for the provided porfolio will be deleted.
+
+### Header
+
+<ParamField header="Authorization" type="string" default="none" required>
+  The authentication token for your request
+</ParamField>
+
+### Body
+
+<ParamField body="group_id" type="string" default="none" required>
+  The id of a group to use for the listing sync operation.
+
+  Used for:
+    - searching for an existing listing record to update
+    - setting group assignment for a newly created listing record
+    - deleting listings in the group that are not provided in the request body
+</ParamField>
+
+<ParamField body="listings" type="[object]" default="none">
+  <Expandable title="listing">
+    <ParamField body="unit_id" type="string" default="none">
+      The id of the unit you want to associate the newly created or updated listing.
+
+      If no unit id is specfified, a new unit record will be created instead.
+    </ParamField>
+
+    <ParamField body="amenities" type="[string]" default="none">
+      A list of amenities for the listing to be created or updated.
+    </ParamField>
+
+    <ParamField body="features" type="[string]" default="none">
+      A list of features for the listing to be created or updated.
+    </ParamField>
+
+    <ParamField body="finish" type="string" default="none">
```

Comment:
also minor and maybe not important but you've wrapped string in [] before but not here

Reply:
Parameters wrapped in [] are a list of whatever is between the []. So for feature, for example it'd be:
```
"feature": [
    "Floor heating",
    "Swimming pool"
]
```
and for finish it'd just be
`"finish": "gold"`

Reply:
ah makes sense!